python/instrumentation/openinference-instrumentation-haystack/pyproject.toml

```diff
@@ -0,0 +1,91 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "openinference-instrumentation-haystack"
+dynamic = ["version"]
+description = "OpenInference Haystack Instrumentation"
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = ">=3.9, <3.13"
+authors = [
+    { name = "OpenInference Authors", email = "oss@arize.com" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+dependencies = [
+    "opentelemetry-api",
+    "opentelemetry-instrumentation",
+    "opentelemetry-semantic-conventions",
+    "openinference-instrumentation>=0.1.7",
+    "openinference-semantic-conventions",
+    "wrapt",
+    "typing-extensions",
+]
+
+
+[project.optional-dependencies]
+instruments = [
+    #"guardrails-ai >= 0.4.5",
```

Comment:
remove comment and add version. you can put >= the version of haystack you implemented on